app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<p style="color: green"><%= notice %></p>
+
+<%= render @recipe %>
+
+<div id="recipe_foods">
+  <%= render "recipe_foods/table_header" %>
+  <% @recipe.recipe_foods.includes(:food).each do |recipe_food| %>
+    <%= render partial: "recipe_foods/recipe_food", locals: {recipe_food: recipe_food} %>
+  <% end %>
+</div>
+
+<div class="show-links">
+  <%= link_to "Edit", edit_user_recipe_path(current_user, @recipe), class:"black-btn" %>
+  <%= link_to "Back", user_recipes_path(current_user), class:"black-btn" %>
+
+  <%= button_to "Delete", user_recipe_path(current_user, @recipe), method: :delete, class:"black-btn" %>
+</div>
```

Comment:
- [ ] Good job so far team 👏 On this page, we are supposed to have the `Add ingredient`  button to help us add food that makes up a particular recipe and the `Generate shopping list` . Currently, I can find it, also please follow the wireframe given when you add that button. Here is the comparison 👇 
**Wireframe**
![image](https://user-images.githubusercontent.com/90222110/219747470-fc091f0c-d440-4be2-9dae-72008bd462f8.png)
**Your design**
![image](https://user-images.githubusercontent.com/90222110/219748996-2876ad6b-1d87-46bb-8653-45ee57ea40ac.png)

Please add the Add ingredient button and make sure that its functionality is working, It allow user to add food in the shown table and the Generate shopping list button which lists food that are on the ingredient table but not in the foods table 😃 👍 
HINT: the `recipe_foods` table will be used